ui/e2e/instances/details/index.spec.ts

```diff
@@ -87,3 +95,156 @@ test("the header of the instance page shows the relevant data for the workflow",
     "the button 'Open Workflow' is clickable and links to the correct URL"
   ).toHaveURL(editURL);
 });
+
+test("the diagram panel on the instance page responds to user interaction", async ({
+  page,
+}) => {
+  const newInstance = createInstance({ namespace, path: simpleWorkflowName });
+  await expect(newInstance, "wait until process was completed").toBeDefined();
+  const instanceId = (await newInstance).instance;
+  await page.goto(`/${namespace}/instances/${instanceId}`);
+
+  const diagramPanel = page.getByTestId("rf__wrapper");
+  await expect(diagramPanel, "It renders the diagram panel").toBeVisible();
+
+  const startNode = diagramPanel.getByTestId("rf__node-startNode");
+  const endNode = diagramPanel.getByTestId("rf__node-endNode");
+
+  await expect(startNode, "It renders the start node").toBeVisible();
+  await expect(endNode, "It renders the end node").toBeVisible();
+
+  const resizeButton = page.getByTestId("resizeDiagram");
+  await expect(resizeButton, "It renders the maximize button").toBeVisible();
+
+  await resizeButton.hover();
+  await expect(
+    page.getByText("maximize diagram"),
+    "it shows the correct text when hovering over the resize button"
+  ).toBeVisible();
+  // hover over something else to make the overlay disappear
+  page.getByTestId("rf__wrapper").hover();
+
+  await resizeButton.click();
+  await expect(
+    diagramPanel,
+    "It maximizes the diagram to full width of the screen"
+  ).toHaveCSS("width", "1032px");
+  await resizeButton.hover();
+
+  await expect(
+    page.getByText("minimize diagram"),
+    "it shows the correct text when hovering over the resize button"
+  ).toBeVisible();
+  // hover over something else to make the overlay disappear
+  page.getByTestId("rf__wrapper").hover();
+
+  await page.reload();
+  await expect(
+    diagramPanel,
+    "After a page reload the setting for maximizing the width is remembered"
+  ).toHaveCSS("width", "1032px");
+
+  await resizeButton.click();
+  await expect(
+    diagramPanel,
+    "It minimizes the diagram to the initial width of the screen"
+  ).toHaveCSS("width", "506px");
+});
+
+test("the diagram on the instance page changes appearance dynamically", async ({
+  page,
+}) => {
+  const newInstance = createInstance({
+    namespace,
+    path: workflowWithDelayName,
+  });
+  await expect(newInstance, "wait until process was completed").toBeDefined();
+  const instanceId = (await newInstance).instance;
+  await page.goto(`/${namespace}/instances/${instanceId}`);
+
+  const diagramPanel = page.getByTestId("rf__wrapper");
+  await expect(diagramPanel, "It renders the diagram panel").toBeVisible();
+
+  const resizeButton = page.getByTestId("resizeDiagram");
+  await expect(resizeButton, "It renders the maximize button").toBeVisible();
+  // resize screen to see the nodes better
+  await resizeButton.click();
```

Comment:
I would just make this one line `await page.getByTestId("resizeDiagram").click();`. Because resizing is not tested here, and it would break anyway if the button would not be found. I's just a small thing but it makes the test a bit more readable

Reply:
Alright 👌